src/cloud/reference/ece-tools-reference.md

```diff
@@ -33,6 +33,9 @@ Available commands:
  build
   build:generate           Generates all necessary files for build stage
   build:transfer           Transfer generated files into init directory
+ cloud
+  cloud:config:create      Create .magento.env.yaml file from provided configuration. Overwrite file if exists.
```

Comment:
The list of options has an inconsistent full stop at the end of each line. Please update to all lines either have a full-stop or not.

Reply:
Thank you for your suggestions, this command list is the output of `php ./vendor/bin/ece-tools list` command, so I need to update into `ece-tools` first.